File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.cc
#include <iostream>
#include <string>
using namespace std;

int main(){
    int n, cnt = 0, c = 1;
    string str;
    
    cin >> n;
    
    for(int i = 0; i < n; i++){
        cin >> str;
        if(str.length() == 1){
            cnt++;
            continue;
        }
        for(int j = 0; j < str.length()-2; j++){
            if(str[j] == str[j+1]) continue;
            for(int k = j+2; k < str.length(); k++){
                if(str[j] == str[k]){
                    c = 0;
                    break;
                }
            }
        }
        if(c == 1) cnt++;
        else c = 1;
    }
    cout << cnt;
}

File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.swift
let n = Int(readLine()!)!
var cnt: Int = 0

for _ in 0..<n {
    let input = readLine()!
    var arr: [Character] = []
    
    for i in input {
        if arr.isEmpty {
            arr.append(i)
        } else if arr.last! == i {
            continue
        } else {
            arr.append(i)
        }
    }
    
    if arr.count == Set(arr).count {
        cnt += 1
    }
}

print(cnt)